### PR TITLE
Improve PerMutator logger

### PR DESCRIPTION
**Pull request by theofidry**
- Indicate the score units (%)
- Round and format the scores to ensure they are 2 significant digit instead of integers
- Pad the cells to create a more readable table

### CODE REVIEW

**Comment by sanmai** [comment]
What I really would've loved to see in this table, is the average test run time per mutation. Then you can export this table, sort it, and get several "fastest" mutation, which is a great thing when you want to add mutation testing to some elaborate project with lots of test and huge coverage, where running with `@default` takes forever and makes no sense. Once you get it up and running, you can trivially extend the list of used mutations.

It this an idea worthy of a issue, what do you think?

File: src/Logger/PerMutatorLogger.php [hunk 1]
```diff
@@ -58,30 +70,115 @@ public function __construct(MetricsCalculator $metricsCalculator)
...
+        return $sizes;
+    }
+
+    private static function padCell(string $value, int $size, bool $toLeft): string
+    {
+        return str_pad($value, $size, ' ', $toLeft ? STR_PAD_LEFT : STR_PAD_RIGHT);
+    }
+
+    /**
+     * @var int[]
```
Comment (sanmai) [inline src/Logger/PerMutatorLogger.php]:
```suggestion
     * @param int[] $columnSizes
     *
     * @var int[]
```

**Comment by theofidry** [comment]
How would you handle the sorting?

I agree it would be very useful to have a report purely for internal metrics analysis, but I was more thinking of a CSV format rather than text with a pretty table

**Comment by sanmai** [comment]
I'd use Excel or Sheet, this isn't a problem.

**Comment by theofidry** [comment]
But wouldn't it be preferable to import that data Excel/Sheet ready rather than requiring you to copy it over?

**Comment by sanmai** [comment]
Oh, yes, that'd be nicer. CSV or TSV is definitely better. 

TBH I can't remember why we have Markdown here.

**Comment by theofidry** [comment]
Alright let's open an issue for that then :) I'm still not quite sure what some loggers are used for (e.g. #1123) and if we want one to collect internal metrics it might be worth having a discussion about the data collected in there

**Comment by maks-rafalko** [comment]
Sigh.. GitHub should disable the approve button if builds are failing. 

https://travis-ci.org/infection/infection/jobs/658672709#L402

**Comment by theofidry** [comment]
Yeah gonna fix that and the AppVeyor issue as well before merging

**Pull request merged**